src/cartesian/Bar.tsx

```diff
@@ -64,22 +64,28 @@ type RectangleShapeType =
   | RectangleProps
   | boolean;
 
-interface BarProps extends InternalBarProps {
+export interface BarProps extends InternalBarProps {
   className?: string;
+  index?: Key;
   layout?: 'horizontal' | 'vertical';
   xAxisId?: string | number;
   yAxisId?: string | number;
   stackId?: string | number;
   barSize?: number;
   unit?: string | number;
-  name?: string | number;
```

Comment:
is this removal safe? I wouldn't ever expect `name` to be a number but downstream consumers could have it as a number? this would break anyone that does (unfortunately).

https://recharts.org/en-US/api/Bar
Old docs still say `String | Number` and same with the rest of the cartesian components. I think we should keep this for now

Reply:
Fixed